dsn.go

```diff
@@ -65,6 +66,8 @@ type Config struct {
 	MultiStatements          bool // Allow multiple statements in one query
 	ParseTime                bool // Parse time values to time.Time
 	RejectReadOnly           bool // Reject read-only connections
+
+	BeforeConnect func(context.Context, *Config) error // Invoked before a connection is established
```

Comment:
Move this member to right after `Logger`.

Reply:
Done!